javalib/src/main/scala/java/util/concurrent/atomic/LongAdder.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+package java.util.concurrent.atomic
```

Comment:
Our header checker will complain that there should be one empty line between the `*/` and the `package` lines.